### ISSUE TITLE
Child Bridge functionality

### ISSUE THREAD

**Issue by tdoerig** [kdietrich/homebridge-devolo#75]
HI,

Is it possible to add child bridge functionality to this plugin? 
(Homebridge allows you to run your plugin's platform or accessory as an isolated child bridge. This can improve the general responsiveness and reliability of Homebridge. https://github.com/homebridge/homebridge/wiki/Child-Bridges)

Thank you

**Comment by gwi2014** [comment]
According to the documentation URL you provided this should work without any code change:

> This will work with all existing plugins without requiring any code changes.

**Comment by tdoerig** [comment]
> According to the documentation URL you provided this should work without any code change:
> 
> > This will work with all existing plugins without requiring any code changes.

You are right. The Problem seems to be the following. When I try to open the plugin configuration an error appears (failed to load config: plugin alias could not be deterninded).

**Comment by gwi2014** [comment]
Can you post your configuration? It could be a problem in Homebridge UI.

You can try to add the `_bridge` section manually to your devolo platform.

**Comment by tdoerig** [comment]
{
            "platform": "Devolo",
            "name": "Devolo",
            "host": "10.0.1.22",
            "email": "....@gmail.com",
            "password": "**************",
            "uuid": "**************",
            "gateway": "**************",
            "passkey": "**************",
            "sceneWhitelist": [
                "Guten Morgen",
                "Gute Nacht",
                "Alarmanlage einschalten",
                "Alarmanlage ausschalten"
            ],
            "_bridge": {
                "username": "0E:B8:2B:20:76:36"
            }
        }
